x-pack/test/functional/apps/index_management/index_details_page/index.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default ({ loadTestFile }: FtrProviderContext) => {
```

Comment:
My thinking was to have index details page tests run separately, because they need to set `--xpack.index_management.dev.enableIndexDetailsPage=true` in the config [file](https://github.com/elastic/kibana/pull/163521/commits/49351038966dfac2753d8f450102913ed6037be4#diff-69470c2245c507eeb4fd782987c6e71e94fba90f05dd630840f6e30db267785b). I added this config to the buildkite [file](https://github.com/elastic/kibana/pull/163521/commits/49351038966dfac2753d8f450102913ed6037be4#diff-6eef05803723603412ed7dab4965e81e58ad26b6a2a091b65be6fc392f2d65f2) so that the new tests will run in addition to the normal tests that don't have the config set. 
This will need to be merged together at the end of the project when the feature flag will be removed.

Reply:
Ohh gotcha now! That makes sense!